### ISSUE TITLE
!meetup not configuring

### ISSUE THREAD

**Issue by Codeflare** [FoglyOgly/Meowth#210]
I had read somewhere that !meetup was cross-feeding with the EX Raid stuff, but I can't configure !meetup at all. When I try to, it asks for the channel to hook it to, and does not move forward when I reply with a channel name. If this is intentional while it is fixed, that makes sense. Would just like to know when I can try again.

Thanks!

**Comment by kwilliams147** [comment]
I am having a different issue with meetup configuration. When i try and run !configure and then meetup it tells me it it doesn't understand.

**Comment by nileplumb** [comment]
Yup can't get it to work at all, tried everything I could think of. meetup is also missing from the initial configure set up too.

**Comment by kwilliams147** [comment]
@Codeflare @nileplumb I got this from their discord just now but now when I reply it just hangs up and doesn't respond.

`FoglyOgly - Yesterday at 11:59 AM
meetup is bugged in the dm reply
you have to use !configure meetup and sometimes that doesn't work either`

Update: Talked to FoglyOgly on discord and it is an known issue.

**Comment by scragly** [comment]
Adding an actual response here to confirm the meetup response in configure DMs are a known bug that's been annoyingly persistent in hiding from fogly's debugging. 

In the mean time, as @kwilliams147 has stated, calling `!configure meetup` directly as the initial command will allow you to configure the meetup settings, however this is still considered as an active bug in the configure DM session, so won't be closed until it's resolved.

Thanks for notifying us here as an Issue, and we'll update once a fix is implemented.
